test/parallel/test-fs-rm.js

```diff
@@ -181,6 +232,39 @@ if (isGitPresent) {
     fs.rmSync(filePath, common.mustNotMutateObjectDeep({ force: true }));
   }
 
+  // Should delete a valid symlink
+  const linkTarget = path.join(tmpdir.path, 'link-target.txt');
+  fs.writeFileSync(linkTarget, '');
+  const validLink = path.join(tmpdir.path, 'valid-link');
+  fs.symlinkSync(linkTarget, validLink);
+  try {
+    fs.rmSync(validLink);
```

Comment:
This code doesn't seem to test if `fs.rmSync` succeeds.
```suggestion
    fs.rmSync(validLink);
    assert.strictEqual(fs.existsSync(validLink), false);
```

Reply:
Thanks, done.